src/core/include/openvino/core/attribute_adapter.hpp

```diff
@@ -463,5 +463,6 @@ class OPENVINO_API AttributeAdapter<std::set<std::string>> : public DirectValueA
 public:
     OPENVINO_RTTI("AttributeAdapter<set<string>>");
     AttributeAdapter(std::set<std::string>& value) : DirectValueAccessor<std::set<std::string>>(value) {}
```

Comment:
do you know whether it's possible that we have other `AttributeAdapter` instances which are not defined in this file?

Or each usage of `AttributeAdapter` should be doubled here?